### PR TITLE
added readFromFile method

### PR DESCRIPTION
**Pull request by YarikJ**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -8,7 +14,31 @@
...
+        try {
+            List<String> lines = Files.readAllLines(Paths.get(fileName));
+            if (lines.isEmpty()) {
+                return new String[0];
+            }
+
+            for (String line : lines) {
+                String[] lineWords = line.toLowerCase().split(" ");
+                for (String word : lineWords) {
+                    if (word.charAt(0) == 'w') {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: make 'w' constant of a class

Comment (YarikJ) [reply src/main/java/core/basesyntax/FileWork.java]: done

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -8,7 +14,32 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
-    public String[] readFromFile(String fileName) {
-        return null;
+    public static final char W = 'w';
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: can you make this field name more informative?

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -8,7 +14,32 @@
...
+
+    public static String[] readFromFile(String fileName) {
+        StringBuilder words = new StringBuilder();
+        try {
+            List<String> lines = Files.readAllLines(Paths.get(fileName));
+            if (lines.isEmpty()) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: try to solve without this check

File: src/main/java/core/basesyntax/FileWork.java [hunk 4]
```diff
@@ -8,7 +14,32 @@
...
+
+            for (String line : lines) {
+                String[] lineWords = line.toLowerCase().split(" ");
+                for (String word : lineWords) {
+                    if (word.charAt(0) == W) {
+                        words.append(word.replaceAll("[^a-z]+", "")).append(" ");
+                    }
+                }
+            }
+            if (words.toString().isBlank()) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: try to solve without this check

File: src/main/java/core/basesyntax/FileWork.java [hunk 5]
```diff
@@ -8,7 +15,27 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
-    public String[] readFromFile(String fileName) {
-        return null;
+    public static final char START_LETTER = 'w';
+
+    public static String[] readFromFile(String fileName) {
+        ArrayList<String> result = new ArrayList<>(0);
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]:
```suggestion
        ArrayList<String> result = new ArrayList<>();
```

**Pull request closed without merging**